main.go

```diff
@@ -37,7 +37,6 @@ Open Source, so feel free to contribute !
 	jiraTicketTypePtr := flag.String("jiraTicketType", "Bug", "Optional. Chosen JIRA ticket type")
 	severityPtr := flag.String("severity", "low", "Optional. Your severity threshold")
 	maturityFilterPtr := flag.String("maturityFilter", "", "Optional. include only maturity level(s) separated by commas [mature,proof-of-concept,no-known-exploit,no-data]")
-	priorityScorePtr := flag.Int("priorityScoreThreshold", 0, "Optional. Your min priority score threshold [INT between 0 and 1000]")
```

Comment:
We can't drop that, it has been requested in the past, that's why it got added. Note this is used to filter out issues that are below that number, and has nothing to do with the severity-to-priority mapping